community/socklog/APKBUILD

```diff
@@ -2,48 +2,43 @@
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=socklog
 pkgver=2.1.0
-pkgrel=2
+pkgrel=3
 pkgdesc="System and kernel logging services for runit"
 url="http://smarden.org/socklog/"
 arch="all"
 license="BSD"
 install="$pkgname.post-install"
 subpackages="$pkgname-doc"
-source="http://smarden.org/$pkgname/$pkgname-$pkgver.tar.gz"
-
-_builddir="$srcdir"/admin/socklog-$pkgver/src
-
-prepare() {
-        local i
-        cd "$_builddir"
-        for i in $source; do
-                case $i in
-                *.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-                esac
-        done
-        echo "$CC $CFLAGS" > conf-cc
-        echo "$CC $LDFLAGS" > conf-ld
-}
+source="http://smarden.org/$pkgname/$pkgname-$pkgver.tar.gz
+	socklog.setup
+	"
+builddir="$srcdir"/admin/socklog-$pkgver/src
 
 build() {
-	cd "$_builddir"
+	cd "$builddir"
+	echo "$CC $CFLAGS" > conf-cc
+	echo "$CC $LDFLAGS" > conf-ld
 	make
 }
 
 package() {
-	cd "$_builddir"
+	cd "$builddir"
 	install -d "$pkgdir"/usr/bin
 	install -d "$pkgdir"/sbin
```

Comment:
Since you copy files using `install`, there’s no need to create the directory beforehand. Just add `-D` option, i.e. `install -m755 -D "$i" "$pkgdir"/sbin/$i || return 1`